lib/lens/core.rb

```diff
@@ -3,5 +3,6 @@
 require "lens/sender"
 require "lens/event_formatter"
 require "lens/gc"
+require "lens/event"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.